hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/protocolPB/StorageContainerLocationProtocolClientSideTranslatorPB.java

```diff
@@ -157,6 +158,12 @@ public final class StorageContainerLocationProtocolClientSideTranslatorPB
   private final StorageContainerLocationProtocolPB rpcProxy;
   private final SCMContainerLocationFailoverProxyProvider fpp;
 
+  /**
+   * This is used to check if 'leader' or 'follower' exists,
+   * in order to confirm whether we have enabled Ratis.
+   */
+  private final List<String> scmRatisRolesToCheck = Arrays.asList("leader", "follower");
```

Comment:
The variable `scmRatisRolesToCheck` is only referenced in one place, and I'm not sure if defining it as a global variable is a good idea. Additionally, from my understanding, using `static` might be better if it needs to be used. @ChenSammi  Please help give some suggestions.

Reply:
@errose28 @adoroszlai May I take up some of your time to ask for some advice? Thank you very much!